### DISCUSSION TITLE
Stuck

### DISCUSSION THREAD

**Discussion by Brakisshuro** [jlesage/docker-filebot#98] [General]
Have a Asustor Nas, with docker running, portainer being gui. I have installed and seems to be setup right. Looks correct in env but yet it wont open via browser. I am new to Docker so I am propbably screwing something up, but really want to use filebot to rebuild my library.  my log is attached, what else can I provide?
[_Filebot_logs.txt](https://github.com/jlesage/docker-filebot/files/12413191/_Filebot_logs.txt)
my inspect file - https://pastebin.com/XFK05gLQ

**Comment by jlesage** [comment]
According to the provided log file, you are using an image from Linuxserver.  This project is for the `jlesage/filebot` Docker image.  Which one you want to use?
(by the way, your inspect file is not accessible)
